exercises/ex_stam.py

```diff
@@ -0,0 +1,49 @@
+# Stam Exercise
+
+import sys
+PATH = 1
+FIRST_NAME = "Shira"
+LAST_NAME = "Vaknin"
+PATH_TO_OTHER_FILE = r"Z:\python\my_name.txt"
+
+
+def add_name(file_path, firs_name, last_name, other_path):
+    """
+    Get file address add my name to the line and save it in new location.
+
+    Add my first name to the beginning of each line.
+    Add my last name to the end of each line.
+
+    Args:
+        file_path: a string,
+        firs_name:
+        last_name:
+        other_path:
+    """
+    try:
+
+        with open(file_path, 'r') as file_pointer:
+
+            with open(other_path, 'w') as new_file:
+
+                for line in file_pointer:
+
+                    new_file.write("{} {} {}\n".
+                                   format(firs_name, line.rstrip(), last_name))
+
+        print("new file saved in:" + other_path)
+
+    except Exception as e:
+        print(e)
```

Comment:
if you don't do anything with the exception and just print it, its better to just throw the exception. This way you can "hide" bugs

Reply:
?

Reply:
Please finish the review..........